Python/rerf/rerfClassifier.py

```diff
@@ -189,6 +189,15 @@ def fit(self, X, y):
         X, y = check_X_y(X, y)
         num_features = X.shape[1]
 
+        # Check that labels are starting from 0
+        if min(y) != 0:
+            raise ValueError("Labels must start from 0")
+
+        # Check that labels are an inclusive range
+        y_set = set(y)
+        if len(y_set) - 1 < max(y_set):
+            raise ValueError("Labels must not contain missing values")
```

Comment:
That sounds to me like there may be NA's in the Y's. 
Can we re-word to something like, "Labels must be contiguous from [0, k-1]" ?